gson/pom.xml

```diff
@@ -26,56 +26,21 @@
     <relativePath>../project/pom.xml</relativePath>
   </parent>
 
-  <!-- 
-    JCLOUDS-1166: This project relocates the 'com.google.gson.internal' package
-    so we can keep using its contents in OSGi environments.
-    This is a temporal solution and we should try to remove the usages of the classes in
-    there and remove this once that is done.
-  -->
-  <artifactId>jclouds-gson</artifactId>
-  <name>jclouds gson copy without internal packages</name>
-  <packaging>bundle</packaging>
+  <artifactId>jclouds-gson-modules</artifactId>
```

Comment:
[minor] Just curious what this aggregator POM does? I guess it's because we didn't want two gson-related bundles at the top level (if so, definitely agree that that)?